tests/edge-cases/point_add-06.txt

```diff
@@ -0,0 +1,2 @@
+brun '(point_add (q 0xffff) (q 0xffff))'
+FAIL: point_add expects blob, got 82ffff: Length of bytes object not equal to G1Element::SIZE (0xffff 0xffff)
```

Comment:
it's a little bit odd that the error message prints the *serialized* form of the value it was passed

Reply:
Hmm, that is odd. Probably could do with a realignment.

Reply:
when an `SExp` is printed (via `__str__`) it uses `to_bin()` which in turn uses the serializer. I think it would be better to keep the serialization separate. Ideally we could change the serialization format without changing the way an `SExp` is printed. I will propose a patch for this

Reply:
I've made this change, and removed `to_bin()`. Please take a look